pkg/apiserver/statement/statement.go

```diff
@@ -40,15 +50,23 @@ func NewService(p ServiceParams) *Service {
 
 func Register(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
 	endpoint := r.Group("/statements")
-	endpoint.Use(auth.MWAuthRequired())
-	endpoint.Use(utils.MWConnectTiDB(s.params.TiDBClient))
-	endpoint.GET("/config", s.configHandler)
-	endpoint.POST("/config", s.modifyConfigHandler)
-	endpoint.GET("/time_ranges", s.timeRangesHandler)
-	endpoint.GET("/stmt_types", s.stmtTypesHandler)
-	endpoint.GET("/overviews", s.overviewsHandler)
-	endpoint.GET("/plans", s.getPlansHandler)
-	endpoint.GET("/plan/detail", s.getPlanDetailHandler)
+	{
+		endpoint.GET("/download", s.downloadHandler)
```

Comment:
maybe it's better to register it with other endpoints
```go
endpoint.GET("/plan/detail", s.getPlanDetailHandler)
endpoint.GET("/download", s.downloadHandler)
endpoint.GET("/download/acquire_token", s.downloadTokenHandler)
```

Reply:
It is because the `/download` API doesn't and shouldn't need `auth.MWAuthRequired()` middleware.

Reply:
got